dace/optim/auto_tuner.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
```

Comment:
also missing copyright, see guidelines (and maybe add yourself to AUTHORS while you're at it)